src/tasks/minions/woodcuttingActivity.ts

```diff
@@ -93,6 +93,7 @@ async function handleForestry({ user, duration, loot }: { user: MUser; duration:
 		const event = ForestryEvents[eventIndex];
 		const defaultEventXP = 5 * (randInt(85, 115) / 100); // used for unverified xp rates
 		let eggsDelivered = 0;
+		let beeHive = 0;
```

Comment:
This variable should be given a better name, just reading it, i have 0 idea what it is for or does.

Reply:
I renamed it beeHiveRepairs to more accuratley reflect what it is. It's purpose is to get a more accurate xp rate to ingame xp rates. 

`for (let i = 0; i < randInt(5, 7); i++)`
- accounts for the average of 6 beehives per event, 5-7 for variance depending on how "busy" the event is

`beeHiveRepairs += randInt(5, 10);`
- accounts for the users interaction with repairing the beehive, 5-10 for variance depending on how "busy" the event is

`eventXP[event.uniqueXP] += user.skillLevel(event.uniqueXP) * 0.3 * beeHiveRepairs;`
- beeHiveRepairs is then used here to give an accurate amount of construction xp per event based on the users con level